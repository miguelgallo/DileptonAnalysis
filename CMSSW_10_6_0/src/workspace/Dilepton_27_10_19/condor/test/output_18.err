Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
ImportError: No module named site
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCand
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCandOutOfAccept
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> nGenProtCand
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_status
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_mass
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dpt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dphi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_3Dangle

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00002b431293f41c in waitpid () from /lib64/libc.so.6
#1  0x00002b43128bcf12 in do_system () from /lib64/libc.so.6
#2  0x00002b431221552d in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#3  0x00002b4312217dd4 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#4  <signal handler called>
#5  0x00002b431f42de5c in ?? ()
#6  0xbfcb70a3e0000000 in ?? ()
#7  0xbfb0fd70a0000000 in ?? ()
#8  0xc02a147ae0000000 in ?? ()
#9  0xbfb4bd70a0000000 in ?? ()
#10 0xbff7e147a0000000 in ?? ()
#11 0x00000000005b7ac0 in ?? ()
#12 0x00007ffc07a36310 in ?? ()
#13 0x00002b4311e9c000 in ?? ()
#14 0x00002b431438f520 in cling::Interpreter::RunFunction(clang::FunctionDecl const*, cling::Value*) [clone .part.248] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#15 0x00002b43143906bf in cling::Interpreter::EvaluateInternal(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::CompilationOptions, cling::Value*, cling::Transaction**, unsigned long) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#16 0x00002b4314390c2f in cling::Interpreter::process(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::Value*, cling::Transaction**, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#17 0x00002b431443bb2d in cling::MetaProcessor::process(llvm::StringRef, cling::Interpreter::CompilationResult&, cling::Value*, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#18 0x00002b43142eeb3e in HandleInterpreterException(cling::MetaProcessor*, char const*, cling::Interpreter::CompilationResult&, cling::Value*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#19 0x00002b43142fdeec in TCling::ProcessLine(char const*, TInterpreter::EErrorCode*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#20 0x00002b43120cda4d in TApplication::ProcessLine(char const*, bool, int*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#21 0x00002b4311ce2782 in TRint::ProcessLineNr(char const*, char const*, int*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libRint.so
#22 0x00002b4311ce2ad4 in TRint::HandleTermInput() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libRint.so
#23 0x00002b43122172b8 in TUnixSystem::CheckDescriptors() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#24 0x00002b4312218a98 in TUnixSystem::DispatchOneEvent(bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#25 0x00002b4312135811 in TSystem::Run() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#26 0x00002b43120cbaff in TApplication::Run(bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#27 0x00002b4311ce40fc in TRint::Run(bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libRint.so
#28 0x000000000040097a in main ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00002b431f42de5c in ?? ()
#6  0xbfcb70a3e0000000 in ?? ()
#7  0xbfb0fd70a0000000 in ?? ()
#8  0xc02a147ae0000000 in ?? ()
#9  0xbfb4bd70a0000000 in ?? ()
#10 0xbff7e147a0000000 in ?? ()
#11 0x00000000005b7ac0 in ?? ()
#12 0x00007ffc07a36310 in ?? ()
#13 0x00002b4311e9c000 in ?? ()
#14 0x00002b431438f520 in cling::Interpreter::RunFunction(clang::FunctionDecl const*, cling::Value*) [clone .part.248] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#15 0x00002b43143906bf in cling::Interpreter::EvaluateInternal(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::CompilationOptions, cling::Value*, cling::Transaction**, unsigned long) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#16 0x00002b4314390c2f in cling::Interpreter::process(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::Value*, cling::Transaction**, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#17 0x00002b431443bb2d in cling::MetaProcessor::process(llvm::StringRef, cling::Interpreter::CompilationResult&, cling::Value*, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#18 0x00002b43142eeb3e in HandleInterpreterException(cling::MetaProcessor*, char const*, cling::Interpreter::CompilationResult&, cling::Value*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
===========================================================


