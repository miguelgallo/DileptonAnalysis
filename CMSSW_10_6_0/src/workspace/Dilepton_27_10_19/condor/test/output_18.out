copying file
running
0cortes: 3166960
1cortes: 46903
2cortes: 43488
3cortes: 41657
4cortes: 19414
5cortes: 19412
6cortes: 11105
7cortes: 1530
8cortes: 21
moving file
